util/info.c

```diff
@@ -256,7 +257,9 @@ static int run(struct fi_info *hints, char *node, char *port)
 	uint64_t flags;
 
 	flags = list_providers ? FI_PROV_ATTR_ONLY : 0;
-	ret = fi_getinfo(FI_VERSION(1, 1), node, port, flags, hints, &info);
+	ret = fi_getinfo(FI_VERSION(FI_MAJOR_VERSION,
+				FI_MINOR_VERSION),
```

Comment:
Why is this indented on it's own line? Can't `FI_MINOR_VERSION` be on the line above, or is that above 80 chars?